test/functional/interface_rpc.py

```diff
@@ -21,6 +22,7 @@ def test_getrpcinfo(self):
         command = info['active_commands'][0]
         assert_equal(command['method'], 'getrpcinfo')
         assert_greater_than_or_equal(command['duration'], 0)
+        assert(info['logpath'] == os.path.join(self.nodes[0].datadir, 'regtest', 'debug.log'))
```

Comment:
Use `assert_equal`?

Reply:
Yes, please. Otherwise a failure can not be debugged

Reply:
Fixed